src/js/controller/controller.js

```diff
@@ -361,16 +361,25 @@ Object.assign(Controller.prototype, {
         function _checkPauseOnViewable(model, viewable) {
             const playerState = model.get('state');
             const adState = _getAdState();
+            const playReason = model.get('playReason');
+
+            function pauseWhenVisible() {
```

Comment:
this is super nitpicky but we are pausing when _not_ visible. function name could confuse future contributors

Reply:
Good point, I can change it really quick.